src/screens/OnboardingScreen/OnboardingScreen.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { View, Text, Image, TouchableOpacity, StyleSheet } from 'react-native';
+import Onboarding from 'react-native-onboarding-swiper';
+import styles from './styles';
+
+const Dots = ({selected}) => {
+    let backgroundColor;
+    backgroundColor = selected ? '#51AD28' : 'rgba(0, 0, 0, 0.3)';
+
+    return (
+        <View 
+            style={{
+                width:6,
+                height: 6,
+                marginHorizontal: 3,
+                backgroundColor
+            }}
+        />
+    );
+}
+
+const Skip = ({...props}) => (
+    <TouchableOpacity
+        style={styles.buttonView}
+        {...props}
+    >
+        <Text style={styles.textView}>Skip</Text>
+    </TouchableOpacity>
+);
+
+const Next = ({...props}) => (
+    <TouchableOpacity
+        style={styles.buttonView}
+        {...props}
+    >
+        <Text style={styles.textView}>Next</Text>
+    </TouchableOpacity>
+);
+
+const Done = ({...props}) => (
+    <TouchableOpacity
+        style={styles.buttonView}
+        {...props}
+    >
+        <Text style={styles.textView}>Done</Text>
+    </TouchableOpacity>
+);
+
+const OnboardingScreen = ({navigation}) => {
+    return (
+        <Onboarding
+        SkipButtonComponent={Skip}
+        NextButtonComponent={Next}
+        DoneButtonComponent={Done}
+        DotComponent={Dots}
+        onSkip={() => navigation.replace("Start")}
+        onDone={() => navigation.navigate("Start")}
+        pages={[
+          {
+            backgroundColor: '#fff',
+            image: <Image style={{width: 350, height: 200}} source={require('../../../assets/OB1.png')} />,
```

Comment:
Move the styles to the styles file. We can reuse the same styles here.

Reply:
Done!